lib/fizzy/execute.cpp

```diff
@@ -129,7 +140,11 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 
 Instance instantiate(const Module& module)
 {
-    return {module};
+    // FIXME: set pages from proper section
+    constexpr unsigned pages = 1;
+    // NOTE: fill it with zeroes
+    bytes memory(pages * pagesize, 0);
```

Comment:
Could be
```suggestion
    bytes memory(pages * pagesize);
```

Reply:
Apparently the constructor requires the explicit filling character.

Reply:
not for `bytes`, sorry